src/grafana/deploy/grafana-configuration/pai-serviceview-dashboard.json.template

```diff
@@ -394,93 +394,98 @@
-            "value": "hadoop-data-node"
+            "text": "prometheus",
+            "value": "prometheus"
           },
           {
             "selected": false,
-            "text": "hadoop-jobhistory-service",
-            "value": "hadoop-jobhistory-service"
+            "text": "pylon",
+            "value": "pylon"
           },
           {
             "selected": false,
-            "text": "hadoop-name-node",
-            "value": "hadoop-name-node"
+            "text": "watchdog",
+            "value": "watchdog"
           },
           {
             "selected": false,
-            "text": "hadoop-node-manager",
-            "value": "hadoop-node-manager"
+            "text": "webportal",
+            "value": "webportal"
           },
           {
-            "selected": true,
-            "text": "hadoop-resource-manager",
-            "value": "hadoop-resource-manager"
+            "selected": false,
+            "text": "frameworkcontroller",
+            "value": "frameworkcontroller"
           },
           {
             "selected": false,
-            "text": "job-exporter",
-            "value": "job-exporter"
+            "text": "hivedscheduler",
+            "value": "hivedscheduler"
           },
           {
             "selected": false,
-            "text": "node-exporter",
-            "value": "node-exporter"
+            "text": "framework-watcher_database-controller",
+            "value": "framework-watcher_database-controller"
           },
           {
             "selected": false,
-            "text": "nvidia-drivers",
-            "value": "nvidia-drivers"
+            "text": "write-merger_database-controller",
+            "value": "write-merger_database-controller"
           },
           {
             "selected": false,
-            "text": "prometheus",
-            "value": "prometheus"
+            "text": "poller_database-controller",
+            "value": "poller_database-controller"
           },
           {
             "selected": false,
-            "text": "pylon",
-            "value": "pylon"
+            "text": "log-manager-nginx",
+            "value": "log-manager-nginx"
           },
           {
             "selected": false,
-            "text": "rest-server",
-            "value": "rest-server"
+            "text": "log-manager-logrotate",
+            "value": "log-manager-logrotate"
           },
           {
             "selected": false,
-            "text": "watchdog",
-            "value": "watchdog"
+            "text": "dshuttle-master",
+            "value": "dshuttle-master"
           },
           {
             "selected": false,
-            "text": "webportal",
-            "value": "webportal"
+            "text": "dshuttle-worker",
+            "value": "dshuttle-worker"
           },
           {
             "selected": false,
-            "text": "yarn-exporter",
-            "value": "yarn-exporter"
+            "text": "dshuttle-csi-daemon",
```

Comment:
what if dshuttle is not deployed

Reply:
If so job-exporter will not collect this metrics and we can't see any info about DShuttle in Grafana.